clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -405,6 +405,19 @@
             "iso646jp"
           ]
         },
+        "cobol-lsp.cpy-manager.endevor-dependencies": {
+          "type": "string",
+          "enum": [
+            "ZOWE",
+            "ENDEVOR_PROCESSOR"
+          ],
+          "enumDescriptions": [
+            "ZOWE paths to retrieve copybooks",
+            "ENDEVOR_PROCESSOR to retrieve copybooks"
+          ],
+          "description": "The method that is used to retrieve copybooks for elements open via Explorer for Endevor.",
```

Comment:
"description": "The method that is used to retrieve remote copybooks for elements opened in Explorer for Endevor."